From 38e5750f9a7c02d5b9f695dc90567811878d12b2 Mon Sep 17 00:00:00 2001
From: zhangmeng <zhangmeng.kevin@spacemit.com>
Date: Thu, 14 Mar 2024 18:04:23 +0800
Subject: [PATCH 0482/1204] k1/dts: update dts

1. remove clk_ignore_unused frome bootargs
2. change rdinit=/init.tmp to rdinit=/init

Change-Id: Icc79f3f17122c70d8f9aee79fd49dbbb67f4e559
---
 arch/riscv/boot/dts/spacemit/k1-x_MINI-PC.dts  | 2 +-
 arch/riscv/boot/dts/spacemit/k1-x_deb1.dts     | 2 +-
 arch/riscv/boot/dts/spacemit/k1-x_deb2.dts     | 2 +-
 arch/riscv/boot/dts/spacemit/k1-x_fpga.dts     | 2 +-
 arch/riscv/boot/dts/spacemit/k1-x_fpga_1x4.dts | 2 +-
 arch/riscv/boot/dts/spacemit/k1-x_fpga_2x2.dts | 2 +-
 arch/riscv/boot/dts/spacemit/k1-x_hs450.dts    | 2 +-
 arch/riscv/boot/dts/spacemit/k1-x_kx312.dts    | 2 +-
 arch/riscv/boot/dts/spacemit/k1-x_mingo.dts    | 2 +-
 9 files changed, 9 insertions(+), 9 deletions(-)

diff --git a/arch/riscv/boot/dts/spacemit/k1-x_MINI-PC.dts b/arch/riscv/boot/dts/spacemit/k1-x_MINI-PC.dts
index 7d752223edf8..29145d85f2fb 100644
--- a/arch/riscv/boot/dts/spacemit/k1-x_MINI-PC.dts
+++ b/arch/riscv/boot/dts/spacemit/k1-x_MINI-PC.dts
@@ -109,7 +109,7 @@ dpu_resv: dpu_reserved@27000000 {
 	};
 
 	chosen {
-		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8,initcall_debug=1 clk_ignore_unused rdinit=/init.tmp";
+		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8,initcall_debug=1 rdinit=/init";
 		stdout-path = "serial0:115200n8";
 	};
 
diff --git a/arch/riscv/boot/dts/spacemit/k1-x_deb1.dts b/arch/riscv/boot/dts/spacemit/k1-x_deb1.dts
index 650fd2353e84..02940af4120e 100644
--- a/arch/riscv/boot/dts/spacemit/k1-x_deb1.dts
+++ b/arch/riscv/boot/dts/spacemit/k1-x_deb1.dts
@@ -112,7 +112,7 @@ dpu_resv: dpu_reserved@27000000 {
 	};
 
 	chosen {
-		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8,initcall_debug=1 clk_ignore_unused rdinit=/init.tmp";
+		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8,initcall_debug=1 rdinit=/init";
 		stdout-path = "serial0:115200n8";
 	};
 
diff --git a/arch/riscv/boot/dts/spacemit/k1-x_deb2.dts b/arch/riscv/boot/dts/spacemit/k1-x_deb2.dts
index 02b65426587f..539f4bbe5ca9 100644
--- a/arch/riscv/boot/dts/spacemit/k1-x_deb2.dts
+++ b/arch/riscv/boot/dts/spacemit/k1-x_deb2.dts
@@ -112,7 +112,7 @@ dpu_resv: dpu_reserved@27000000 {
 	};
 
 	chosen {
-		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8,initcall_debug=1 clk_ignore_unused rdinit=/init.tmp";
+		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8,initcall_debug=1 rdinit=/init";
 		stdout-path = "serial0:115200n8";
 	};
 
diff --git a/arch/riscv/boot/dts/spacemit/k1-x_fpga.dts b/arch/riscv/boot/dts/spacemit/k1-x_fpga.dts
index 7729308c822f..ad408810cc9f 100644
--- a/arch/riscv/boot/dts/spacemit/k1-x_fpga.dts
+++ b/arch/riscv/boot/dts/spacemit/k1-x_fpga.dts
@@ -91,7 +91,7 @@ dpu_resv: dpu_reserved@27000000 {
 	};
 
 	chosen {
-		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8,initcall_debug=1 rdinit=/init.tmp";
+		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8,initcall_debug=1 rdinit=/init";
 		stdout-path = "serial0:115200n8";
 	};
 
diff --git a/arch/riscv/boot/dts/spacemit/k1-x_fpga_1x4.dts b/arch/riscv/boot/dts/spacemit/k1-x_fpga_1x4.dts
index b1a7e10704e8..468af06e4c0c 100644
--- a/arch/riscv/boot/dts/spacemit/k1-x_fpga_1x4.dts
+++ b/arch/riscv/boot/dts/spacemit/k1-x_fpga_1x4.dts
@@ -76,7 +76,7 @@ linux,cma@70000000 {
 	};
 
 	chosen {
-		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8,initcall_debug=1 rdinit=/init.tmp";
+		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8,initcall_debug=1 rdinit=/init";
 		stdout-path = "serial0:115200n8";
 	};
 
diff --git a/arch/riscv/boot/dts/spacemit/k1-x_fpga_2x2.dts b/arch/riscv/boot/dts/spacemit/k1-x_fpga_2x2.dts
index d51160756c92..84b047d9b020 100644
--- a/arch/riscv/boot/dts/spacemit/k1-x_fpga_2x2.dts
+++ b/arch/riscv/boot/dts/spacemit/k1-x_fpga_2x2.dts
@@ -76,7 +76,7 @@ linux,cma@70000000 {
 	};
 
 	chosen {
-		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8,initcall_debug=1 rdinit=/init.tmp";
+		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8,initcall_debug=1 rdinit=/init";
 		stdout-path = "serial0:115200n8";
 	};
 
diff --git a/arch/riscv/boot/dts/spacemit/k1-x_hs450.dts b/arch/riscv/boot/dts/spacemit/k1-x_hs450.dts
index 8413a8e323f0..d48bc8b56d64 100644
--- a/arch/riscv/boot/dts/spacemit/k1-x_hs450.dts
+++ b/arch/riscv/boot/dts/spacemit/k1-x_hs450.dts
@@ -110,7 +110,7 @@ dpu_resv: dpu_reserved@27000000 {
 	};
 
 	chosen {
-		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8,initcall_debug=1 clk_ignore_unused rdinit=/init.tmp";
+		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8,initcall_debug=1 rdinit=/init";
 		stdout-path = "serial0:115200n8";
 	};
 
diff --git a/arch/riscv/boot/dts/spacemit/k1-x_kx312.dts b/arch/riscv/boot/dts/spacemit/k1-x_kx312.dts
index f5dd2dd78490..134c3a7cf8f7 100644
--- a/arch/riscv/boot/dts/spacemit/k1-x_kx312.dts
+++ b/arch/riscv/boot/dts/spacemit/k1-x_kx312.dts
@@ -109,7 +109,7 @@ dpu_resv: dpu_reserved@27000000 {
 	};
 
 	chosen {
-		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8,initcall_debug=1 clk_ignore_unused rdinit=/init.tmp";
+		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8,initcall_debug=1 rdinit=/init";
 		stdout-path = "serial0:115200n8";
 	};
 
diff --git a/arch/riscv/boot/dts/spacemit/k1-x_mingo.dts b/arch/riscv/boot/dts/spacemit/k1-x_mingo.dts
index d883d797fde9..2281edbbd5c0 100644
--- a/arch/riscv/boot/dts/spacemit/k1-x_mingo.dts
+++ b/arch/riscv/boot/dts/spacemit/k1-x_mingo.dts
@@ -108,7 +108,7 @@ dpu_resv: dpu_reserved@27000000 {
 	};
 
 	chosen {
-		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8,initcall_debug=1 clk_ignore_unused rdinit=/init.tmp";
+		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8,initcall_debug=1 rdinit=/init";
 		stdout-path = "serial0:115200n8";
 	};
 
-- 
2.47.0

